tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/openwire/amq/JmsCreateConsumerInOnMessageTest.java

```diff
@@ -84,7 +85,7 @@ public void onMessage(Message message) {
          }
       } catch (Exception ex) {
          ex.printStackTrace();
-         assertTrue(false);
+         fail();
```

Comment:
This is entirely broken; since it is inside onMessage, it specifically isnt on the test thread and as a result cant fail the test directly.

It would need to set a marker (e.g atomic ref to the exception), notify the lock, and then check the marker on the test thread to have it fail.